tests/e2e/page.py

```diff
@@ -319,13 +320,29 @@ def wait_for_table_value(self, table: int, row: int, column: int, text: str) ->
             )
         )
 
-    def get_table_value(self, index: int) -> str:
-        return self._driver.find_element(by=By.XPATH, value=f"//tr/td[{index}]").text
+    def get_table_headers(self) -> Mapping[str, int]:
```

Comment:
`dict` could be used instead of `Mapping`.

Reply:
That's right. Changed.